packages/sdk/governance/src/proposals.ts

```diff
@@ -79,13 +59,21 @@ export interface ProposalTransactionJSON {
   value: string
 }
 
-const isRegistryRepoint = (tx: ProposalTransactionJSON) =>
-  tx.contract === 'Registry' && tx.function === 'setAddressFor'
+export type ExternalProposalTransactionJSON = Omit<ProposalTransactionJSON, 'contract'> & {
+  contract?: string
+}
+
+export const isRegistryRepoint = (
```

Comment:
these types were to make the functions compatible with both `ExternalProposalTransactionJSON` and `ProposalTransactionJSON` that is when it is a core contract or just a regular contract with an address. by only requiring the fields used the differences dont matter